src/protean/core/queryset.py

```diff
@@ -176,6 +176,46 @@ def update(self, *data, **kwargs):
 
         return updated_item_count
 
+    def raw(self, query_string: str):
```

Comment:
Why should this be just a string, for example it could be something like a Elastic Search DSL or SQL Alchemy query

Reply:
Won't they be strings as well? Can you give me examples of those?

Reply:
hmmm maybe you are right, but it can be a dict too because both elasticsearch and mongo have dicts for raw queries

Reply:
1. If it's just a dict, then we can allow plugins to parse and reinterpret the query, like what we have done in Dict Repo.

2. If you take ES or Mongo, raw queries can be more involved than just dicts.

Take Mongo itself for example. They usually have a syntax that also invokes the db with the dict, like `db.users.find( { ‘name’ : ‘John’} )`. Then you are left with possibly using something like an eval or execute function, with utilities like this: https://github.com/EqualExperts/mongo-shell-like-query

So it's best to keep this to the lowest common denominator and build from there. If your concern is only about the type hint, we can change it to "Any" now, or we can do it as we add more plugins, because we will be more informed of choices.

Reply:
Hmmm yeah I was just thinking of the type hint, better to let it be Any so that the plugin can decide what it is.